org.eclipse.dawnsci.nexus.test/src/org/eclipse/dawnsci/nexus/file/NexusFileRemoveNodeTest.java

```diff
@@ -23,6 +23,7 @@
 
 	private static String testScratchDirectoryName;
 	private static String FILE_NAME;
+	private static String FILE_NAME_SWMR;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "FILE_NAME_SWMR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)